January Release notes (#10074)
organicmaps/organicmaps: iphone/metadata/en-US/release_notes.txt

• New OpenStreetMap data as of January 8

> • A standalone Track Recording button

Reviewer: • Track Recording Live Activity Widget with the track statistics (iOS16.2 and later)

> • Save recorded tracks automatically with an option to delete later

Reviewer: This one isn't implemented yet. The old confirmation alert is not changed
```suggestion
```

> • Tap tracks to select them and display stats

Reviewer: Maybe move this one before the track recording features? It is more big and useful IMO

Reviewer: Maybe:
```suggestion
• Tap tracks to display statistics and interactive elevation profile
```

And move this one before the track recording features? It is more big and useful IMO

> • Added interactive track elevation profiles

Reviewer: the ele profile is accessible only in the TrackPP so it should be removed 
```suggestion
```
